src/mm2_player.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include "mm2_common.h"
+
+namespace MM2
+{
+    // Forward declarations
+    class mmInfoBase;
+    class mmPlayerData;
+
+    // External declarations
+
+
+    // Class definitions
+    class mmInfoBase {
+    private:
+        void* vtable;
```

Comment:
Could you implement the proper virtual functions instead? There's only two of them.